app/models/form_submission_attempt.rb

```diff
@@ -25,15 +25,31 @@ class FormSubmissionAttempt < ApplicationRecord
 
     event :fail do
       after do
+        user_uuid = form_submission.user_account.id
+        form_type = form_submission&.form_type
+        log_info = { form_submission_id:, benefits_intake_uuid: form_submission&.benefits_intake_uuid, form_type:,
```

Comment:
Will `form_submission` ever be nil?

Reply:
Is the safe navigation operator needed then?

Reply:
nope, I think that was a copy/paste, addressed in 404d8866152ae11f34f5f1f5b9c015efd29430a2